examples/vision/bit.py

```diff
@@ -0,0 +1,308 @@
+"""
+Title: Image Classification using BigTransfer(BiT)
+Author: [Sayan Nath](https://twitter.com/sayannath2350)
+Date created: 2021/08/19 
+Last modified: 2021/08/19
+Description: BigTransfer (BiT) State-of-the-art transfer learning for image classification.
+"""
+
+"""
+## Introduction
+
+
+BigTransfer known as BiT is a State-of-the-art transfer learning method for Image Classification Tasks. a set of pre-trained image models that can be transferred to obtain excellent performance on new datasets, even with only a few examples per class. BiT performs well across a surprisingly wide range of data regimes — from 1 example per class to 1 M total examples. BiT achieves 87.5% top-1 accuracy on ILSVRC-2012, 99.4% on CIFAR-10, and 76.3% on the 19 task Visual Task Adaptation Benchmark (VTAB). On small datasets, BiT attains 76.8% on ILSVRC-2012 with 10 examples per class, and 97.0% on CIFAR-10 with 10 examples per class. A detailed analysis of the main components are described [here](https://arxiv.org/pdf/1912.11370.pdf) which leads to high transfer performance.
+"""
+
+"""
+## Setup
+"""
+
+import re
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+from imutils import paths
+from pprint import pprint
+from collections import Counter
+from sklearn.preprocessing import LabelEncoder
+
+import tensorflow as tf
+from tensorflow import keras
+import tensorflow_hub as hub
+from tensorflow.keras.utils import to_categorical
```

Comment:
You can access it via `keras.utils.to_categorial`, avoid importing individual symbols.

Reply:
No, what I mean is do `from tensorflow import keras` then use `keras.utils.to_categorical(...)`